phoenix-assembly/pom.xml

```diff
@@ -132,6 +132,10 @@
       <groupId>org.apache.phoenix</groupId>
       <artifactId>phoenix-client-${hbase.suffix}</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.apache.phoenix</groupId>
+      <artifactId>phoenix-client-embedded-${hbase.suffix}</artifactId>
+    </dependency>
```

Comment:
I don't think we should to add the standard (old) client to the assembly after this change.